bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -133,7 +155,11 @@ public String getFormData() {
     @JsonIgnore
     public String getEncodedCurrentPagePath() {
         if (getCurrentPage() != null) {
-            return ComponentUtils.getEncodedPath(getCurrentPage().getPath());
+            if (GuideWCMUtils.isForms(getCurrentPage().getPath())) {
+                return ComponentUtils.getEncodedPath(getCurrentPage().getPath());
```

Comment:
Would we be supporting both path's (ie) form path and form container path ?

Reply:
right now we are only differentiating based on forms and sites, from next release these checks will go away and we will only be having form container path and sites container path.